src/medley/core.cljc

```diff
@@ -658,3 +658,10 @@
   {:added "1.4.0"}
   [x]
   (instance? #?(:clj java.util.regex.Pattern :cljs js/RegExp) x))
+
+(defn index-of
+  "Returns the index of the first occurrence of x in a sequential collection, coll, or -1 if not found."
+  [^java.util.List xs x]
```

Comment:
I haven't used ClojureScript in a long time, but it seems that it works even with the advanced level of optimization. I tested this by replacing :simple with :advanced in project.clj.